lego/components.py

```diff
@@ -70,36 +69,33 @@ class RPyCComponent(BaseComponent):
     python on the component.
     """
 
-    def __init__(self, hostname: str, connection: BaseConnection) -> None:
+    _connection: RPyCConnection
+
+    def __init__(self, hostname: str, username: Optional[str], password: Optional[str]) -> None:
```

Comment:
Missing default values
```suggestion
    def __init__(self, hostname: str, username: Optional[str]=None, password: Optional[str]=None) -> None:
```

Reply:
Fixed 👍